### ISSUE TITLE
How to produce the class semantic vector?

### ISSUE THREAD

**Issue by umdreamer** [sounakdey/doodle2search#15]
As I got through the code, and tried to run the experiments. One of the key problems is to produce the class semantic vector using the word2vec or other NLP method. But the problem is that there are some class words that do not exist in the google-news-300. Even there are some simimlar ones, but I can not find how to convert these similar ones into google-news-300 format. For example, I can not find the "axe" in google-news-300 in any form. Although the authors provided the genereted semantic labels (in word2vec), we still confuse on how to generate it.

@sounakdey , do you please give us a hint on how did you deal with these problem? Thank you very much.

**Comment by XDUSPONGE** [comment]
hello,  have you solved the problem
I have the same problem too
I can run this code in pycharm with 
```python
google_300_corpus = api.load("word2vec-google-news-300")
 model = Word2Vec(google_300_corpus)
```
But when I run it in terminal 
It can not run, I changed the code as you said
```python
google_300_corpus = api.load("word2vec-google-news-300")
 model = google_300_corpus
```
And the problem arises, such as axe not in vocabulary

**Comment by umdreamer** [comment]
No, I haven't solved the problem. I use the similar word to replace the original one. I can run the code, but I haven't reproduced the results.
